src/vs/workbench/browser/actions/developerActions.ts

```diff
@@ -276,6 +285,21 @@ class ToggleScreencastModeAction extends Action2 {
 				}
 
 				if (!configurationService.getValue('screencastMode.onlyKeyboardShortcuts') || !titleLabel || shortcut?.commandId && (format === 'keys' || format === 'commandAndKeys' || format === 'commandWithGroupAndKeys')) {
+					// Fix label for arrow keys
+					if (keyLabel) {
+						if (keyLabel.includes('UpArrow')) {
+							keyLabel = keyLabel.replace('UpArrow', '↑');
+						}
+						if (keyLabel.includes('DownArrow')) {
+							keyLabel = keyLabel.replace('DownArrow', '↓');
+						}
+						if (keyLabel.includes('LeftArrow')) {
+							keyLabel = keyLabel.replace('LeftArrow', '←');
+						}
+						if (keyLabel.includes('RightArrow')) {
+							keyLabel = keyLabel.replace('RightArrow', '→');
+						}
+					}
```

Comment:
This is what I meant:

```suggestion
					keyLabel = keyLabel?.replace('UpArrow', '↑')
						?.replace('DownArrow', '↓')
						?.replace('LeftArrow', '←')
						?.replace('RightArrow', '→');
```

Reply:
I'd say your suggestion looks quite fine and I would love to have this feature. the repetition is quite annoying. Please merge and add it to the next release! :)

Reply:
@joaomoreno 

Ah, ok. Your suggested change looks better written than mine. 

What is the next step? Can you make this change or do I need to do something more?

Reply:
@webarthur you should see a button "commit suggestion": https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request